packages/ui/src/stack/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { forwardRef, Children, cloneElement, isValidElement } from 'react';
 import { Flex } from '../flex';
-import { Box } from '../box';
+import { Box, SpacingProps } from '../box';
```

Comment:
above comment: seems like this should not be taken from box, but a space theme file/type def. What do you think?